contracts/base/PeripheryPayments.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {IERC20Minimal} from "@uniswap/v4-core/contracts/interfaces/external/IERC20Minimal.sol";
+
+import {IPeripheryPayments} from "../interfaces/IPeripheryPayments.sol";
+
+import "../libraries/TransferHelper.sol";
+
+abstract contract PeripheryPayments is IPeripheryPayments {
+    /// @inheritdoc IPeripheryPayments
+    function sweepToken(address token, uint256 amountMinimum, address recipient) public payable override {
+        uint256 balanceToken = IERC20(token).balanceOf(address(this));
+        require(balanceToken >= amountMinimum, "Insufficient token");
+
+        if (balanceToken > 0) {
+            TransferHelper.safeTransfer(IERC20Minimal(token), recipient, balanceToken);
```

Comment:
added the IERC20Minimal casting here

Reply:
Probably neater to us "using"
So at the top of the contract you write
```
using TransferHelper for IERC20Minimal
```

and then this line can be
```
token.safeTransfer(recipient, balanceToken
```